### PR TITLE
Webpack assets path

### PR DESCRIPTION
**Pull request by sareh**
Part of https://github.com/bbc/simorgh-infrastructure/issues/189

The base path for the JS files on the Test environment should be `www.test.bbc.com` rather than `www.test.bbc.co.uk`, which it is due to the code being a relative path. 

- Adds linting to Webpack files
- Enables base path for the JS assets to be changed via the env variable `SIMORGH_BASE_PATH`

To test: locally, change the `SIMORGH_BASE_URL` value to be `www.test.bbc.com` and the run `npm run build && npm run start` 
You should see these files in the network tab:

screen shot of dev tools network tab

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by ChrisBAshton** [comment]
@sareh does this resolve https://github.com/bbc/simorgh-infrastructure/issues/189 ?

**Comment by sareh** [comment]
@ChrisBAshton Yes, sorry I linked to the wrong repo.

File: webpack.config.client.js [hunk 1]
```diff
@@ -25,7 +26,9 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       path: resolvePath('build/public'),
       filename: 'static/js/[name].[hash:8].js',
       // need full URL for dev server & HMR: https://github.com/webpack/docs/wiki/webpack-dev-server#combining-with-an-existing-server
-      publicPath: IS_PROD ? `/` : `http://localhost:${webpackDevServerPort}/`,
+      publicPath: IS_PROD
+        ? `${process.env.SIMORGH_BASE_URL}/`
```
Comment (ChrisBAshton) [inline webpack.config.client.js:30]: Ahhh, that's where it is. Thanks 👍

File: webpack.config.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
```
Comment (ChrisBAshton) [inline webpack.config.js]: Can we not comma-separate these instead of having two lines?

**Review by ChrisBAshton — DISMISSED** [review]
Thanks for the quick work, Sareh 👍 

Worth adding to the PR description that this PR also makes sure we lint our Webpack configs.

**Comment by sareh** [comment]
@ChrisBAshton We will also need to update the `.env` file in the Jenkins environment to use `SIMORGH_BASE_URL=https://www.test.bbc.com`

**Review by ChrisBAshton — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Review by bcmn — APPROVED** [review]

**Comment by ChrisBAshton** [comment]
@sareh can we write a test in Simorgh to prevent this from cropping up again?

**Comment by jamesbrumpton** [comment]
We're going to merge this and test with a follow up PR.

**Pull request merged**